src/main/java/com/nike/cerberus/auth/connector/okta/OktaAuthConnector.java

```diff
@@ -62,19 +64,28 @@ public AuthResponse authenticate(String username, String password) {
                 .setUsername(userLogin);
         final AuthResponse authResponse = new AuthResponse().setData(authData);
 
-        final List<Factor> factors;
         if (StringUtils.equals(authResult.getStatus(), OktaClientResponseUtils.AUTHENTICATION_MFA_REQUIRED_STATUS) ||
                 StringUtils.equals(authResult.getStatus(), OktaClientResponseUtils.AUTHENTICATION_MFA_ENROLL_STATUS)) {
 
             authData.setStateToken(authResult.getStateToken());
             authResponse.setStatus(AuthStatus.MFA_REQUIRED);
 
-            factors = oktaClientResponseUtils.getUserFactorsFromAuthResult(authResult);
+            final List<Factor> factors = oktaClientResponseUtils.getUserFactorsFromAuthResult(authResult)
+                    .stream()
+                    // Filter out Okta push, call, and sms because we don't currently support them.
+                    .filter(factor -> {
+                        String type = factor.getFactorType().toLowerCase();
+                        String provider = factor.getProvider();
+                        return ! (provider.equalsIgnoreCase("okta") &&
+                                ImmutableSet.of("push", "call", "sms").contains(type));
```

Comment:
This ImmutableSet is better as a `private static final` on the class with a descriptive name, e.g `UNSUPPORTED_OKTA_MFA_TYPES`